packages/calcite-components/src/components/modal/modal.scss

```diff
@@ -287,7 +287,8 @@ slot[name="primary"] {
         max-block-size: unset;
       }
     }
-    :host([width-scale="#{$size}"][docked]) .container {
+    :host([width-scale="#{$size}"][docked]) .container,
```

Comment:
For consistency, can this use `.width`? Maybe something like this (⚠️ untested ⚠️)?

```scss
:host([docked]) .container .width-#{$size} {
  // ...
}
```